TECA install target doesn't find teca_py_core.py (etc) in source directory

From the TECA_3rdparty enchilada build:

CMake Error at core/cmake_install.cmake:77 (file):
  file INSTALL cannot find
  "/Users/jeff/projects/TECA_3rdparty/TECA/core/teca_py_core.py".
Call Stack (most recent call first):
  cmake_install.cmake:36 (include)

teca_py_core.py does exist in the build directory. Should we just install these kinds of files from there instead of looking for them in the source directories?

--- a/CMake/teca_python.cmake
+++ b/CMake/teca_python.cmake
@@ -59,7 +59,7 @@ function(teca_python_module mname)
             ${CMAKE_CURRENT_BINARY_DIR}/teca_py_${mname}.py
             ${CMAKE_CURRENT_BINARY_DIR}/../lib)
     install(TARGETS _teca_py_${mname} LIBRARY DESTINATION lib)
-    install(FILES teca_py_${mname}.py DESTINATION lib)
+    install(FILES ${CMAKE_CURRENT_BINARY_DIR}/teca_py_${mname}.py DESTINATION lib)
 endfunction()
 configure_file(teca.py ${CMAKE_CURRENT_BINARY_DIR}/lib/teca.py COPYONLY)
 install(FILES teca.py DESTINATION lib)

